base/applications/taskmgr/perfdata.c

```diff
@@ -125,14 +128,22 @@ static void SidToUserName(PSID Sid, LPWSTR szBuffer, DWORD BufferSize)
 VOID
 WINAPI
 CachedGetUserFromSid(
-    PSID pSid,
-    LPWSTR pUserName,
-    PULONG pcwcUserName)
+    _In_opt_ PSID pSid,
+    _Out_ LPWSTR pUserName,
+    _Inout_ PULONG pcwcUserName)
 {
     PLIST_ENTRY pCur;
     PSIDTOUSERNAME pEntry;
     ULONG cbSid, cwcUserName;
 
+    if (!pSid)
+    {
+        DPRINT("No pSid\n");
```

Comment:
The only way for this to become null is if `PerfDataInitialize` fails to allocate memory. So it would make more sense to print there (once) than here (every loop).

Reply:
`PerfDataInitialize()` is (already) taken care of on #2946.
`GetSecurityInfo()` documentation says it could fail too: I'll move this trace there...